tests/unitest/unitest_test.go

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package unitest
+
+import (
+        "time"
+	"regexp"
+
+        . "github.com/kubearmor/karts/util"
```

Comment:
```suggestion
        . "github.com/kubearmor/KubeArmor/tests/util"
```

Need to update the package name due to #916